plugins/caddy/README.md

```diff
@@ -354,6 +354,7 @@ What does these directives mean?
 |  Key                                      |  Description                                                                                                                                 |  Value example                                                                                                          |
 |:------------------------------------------|:---------------------------------------------------------------------------------------------------------------------------------------------|:------------------------------------------------------------------------------------------------------------------------|
 | `allowed_http_verbs`                      | The HTTP verbs allowed to be cached                                                                                                          | `GET POST PATCH`<br/><br/>`(default: GET HEAD)`                                                                         |
+| `allowed_addional_status_code`            | The additional HTTP status codes allowed to be cached                                                                                        | `202 400`                                                                         |
```

Comment:
Should this be `allowed_additional_status_codes` ?

Reply:
Gotcha! 😄